sample_app/app_data.c

```diff
@@ -157,6 +159,7 @@ uint8_t * app_data_get_input_data (
    }
 
    /* Automated RT Tester scenario 2 - unsupported (sub)module */
+   *size = 0;
```

Comment:
Borde fungera. Kan också vara så att vi måste sätta data för att stacken skall acceptera uppkopplingen. Vi bör dubbelkolla scenario 2 innan vi checkar in.